internal/policy/policy.go

```diff
@@ -695,3 +694,17 @@ func (s *State) findDelegationEntry(roleName string) (tuf.Delegation, error) {
 		}
 	}
 }
+
+func ListRules(ctx context.Context, repo *git.Repository) ([]tuf.Delegation, error) {
+	state, err := LoadCurrentState(ctx, repo)
+	if err != nil {
+		return nil, err
+	}
+
+	targetsMetadata, err := state.GetTargetsMetadata(TargetsRoleName)
```

Comment:
We need to explore delegated roles as well.

Reply:
Added